### ISSUE TITLE
Error to start / reload the plugin

### ISSUE THREAD

**Issue by crashzk** [TheByKotik/sbpp_discord#7]
I am getting the following error after I updated to the latest version of [release](https://github.com/TheByKotik/sbpp_discord/releases) @TheByKotik :
```
L 01/23/2021 - 12:58:21: [SM] Exception reported: Array index out-of-bounds (index -1377762427)
L 01/23/2021 - 12:58:21: [SM] Blaming: sbpp_discord.smx
L 01/23/2021 - 12:58:21: [SM] Call stack trace:
L 01/23/2021 - 12:58:21: [SM]   [1] Line 336, D:\Coding\SourcePawn\sbpp_discord\sourcemod\scripting\sbpp_discord.sp::Resolve
L 01/23/2021 - 12:58:21: [SM]   [2] Line 338, D:\Coding\SourcePawn\sbpp_discord\sourcemod\scripting\sbpp_discord.sp::Resolve
L 01/23/2021 - 12:58:21: [SM]   [3] Line 276, D:\Coding\SourcePawn\sbpp_discord\sourcemod\scripting\sbpp_discord.sp::Settings_Reload
L 01/23/2021 - 12:58:21: [SM]   [4] Line 59, D:\Coding\SourcePawn\sbpp_discord\sourcemod\scripting\sbpp_discord.sp::OnPluginStart
L 01/23/2021 - 12:58:21: [SM] Unable to load plugin "sbpp_discord.smx": Error detected in plugin startup (see error logs)
```

**Comment by crashzk** [comment]
I turned the servers off and on and it was normal. I will be doing tests to see if everything is right, any problems will return here again, for now I will close the issues.

**Comment by TheByKotik** [comment]
Must be fixed in 2c1f7e1.
